conductor/submitter_maya.py

```diff
@@ -148,9 +148,25 @@ def applyDefaultSettings(self):
         super(MayaConductorSubmitter, self).applyDefaultSettings()
         start, end = maya_utils.get_frame_range()[0]
         self.setFrameRange(start, end)
+        self.configureGPUWidget()
         self.extended_widget.refreshUi()
         self.setOutputDir(maya_utils.get_image_dirpath())
 
+    def configureGPUWidget(self):
+        '''
+        Check if V-Ray is current renderer is set to a GPU mode, if so
+        then show the GPU options in submitter
+        '''
+        if maya_utils.is_vray_renderer():
```

Comment:
Can you combine the two `if` statements into one?
eg. `if maya_utils.is_vray_renderer() and maya_utils.is_vray_gpu_enabled()` to minimize the need for indentation?